README.md

```diff
@@ -373,6 +373,50 @@ docker run -d -p 80:80 -p 443:443 \
 
 You'll need apache2-utils on the machine where you plan to create the htpasswd file. Follow these [instructions](http://httpd.apache.org/docs/2.2/programs/htpasswd.html)
 
+### Upstream (Backend) Server HTTP Load Balancing Support
+
+> **Warning**
+> This feature is experimental.  The behavior may change (or the feature may be removed) without warning in a future release, even if the release is not a new major version.  If you use this feature, please provide feedback so we know whether it is working properly and can be promoted to officially supported.
+To enable HTTP load balancing between `nginx-proxy` and a group of backend servers, set the `com.github.nginx-proxy.nginx-proxy.loadbalance` label on the server's container to the load balancing algorithm which should be used. See the [nginx load balancing documentation](https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/) and the [Docker label documentation](https://docs.docker.com/config/labels-custom-metadata/) for details.
+The load balancing algorithm is enabled for the containers sharing the same `VIRTUAL_HOST`
+#### Choosing a Load-Balancing Method
```

Comment:
I don't think it's necessary to go into detail about choosing a load balancing method.  I think it is sufficient to just link to the nginx documentation and explain how to set up load balancing in nginx-proxy.  Something like this:

> ### Upstream (Backend) Server HTTP Load Balancing Support
>
>> **Warning**
>> *(experimental warning goes here)*
>
> If you have multiple containers with the same `VIRTUAL_HOST` and `VIRTUAL_PATH` settings, nginx will spread the load across all of them.  To change the load balancing algorithm from nginx's default (round-robin), set the `com.github.nginx-proxy.nginx-proxy.loadbalance` label on one or more of your application containers to the desired load balancing directive.  See the [`ngx_http_upstream_module` documentation](https://nginx.org/en/docs/http/ngx_http_upstream_module.html) for available directives.
>
>> **Note**
>>   * Don't forget the terminating semicolon (`;`).
>>   * If you are using Docker Compose, remember to escape any dollar sign (`$`) characters (`$` becomes `$$`).
>
> Docker Compose example:
>
> ```yaml
> services:
>   nginx-proxy:
>     image: nginxproxy/nginx-proxy
>     ports:
>       - "80:80"
>     volumes:
>       - /var/run/docker.sock:/tmp/docker.sock:ro
>     environment:
>       HTTPS_METHOD: nohttps
>   myapp:
>     image: jwilder/whoami
>     expose:
>       - "8000"
>     environment:
>       VIRTUAL_HOST: myapp.example
>       VIRTUAL_PORT: "8000"
>     labels:
>       com.github.nginx-proxy.nginx-proxy.loadbalance: "hash $$remote_addr;"
>     deploy:
>       replicas: 4
> ```